### DISCUSSION TITLE
[FEATURE] Library for integration into Source engine games

### DISCUSSION THREAD

**Discussion by chexo3** [ArgosOfIthica/scalu#47] [Ideas]
**Is your feature request related to a problem? Please describe.**
Scalu is useful for making maintenance of cfg files easier. It might be easier to just parse Scalu files directly in engine.

**Describe the solution you'd like**
From what I recall Python to C/C++ bindings exist already. Scalu could maybe be implemented as part of a clientside scripting solution that would include being able to run Python. Maybe like an extra language for VScript?

**Describe alternatives you've considered**
It might make more sense to write a language from scratch that can take advantage of the reduced limitations we have from being able to interface with the engine directly.

**Comment by ArgosOfIthaca** [comment]
I can certainly attempt to help you with an initiative like this, but you'd need to lead it. My actual experience with directly interfacing with Source is very limited, and the time I currently have to pour into big additions like this even more so.
